portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Date;
+
+import com.google.sps.data.Comment;
+import com.google.sps.data.CommentBuilder;
+
+import com.google.gson.Gson;
+
+@WebServlet("/comments")
+public final class CommentsServlet extends HttpServlet {
+    private List <Comment> getCommentsList() {
+        ArrayList <Comment> commentsList = new ArrayList <Comment>();
+
+        commentsList.add(new CommentBuilder()
+            .setId(1)
+            .setName("Max B")
+            .setDescription("My prev comment!")
+            .setPostDate(new Date(1595318478000L))
+            .build()
+        );
+
+        commentsList.add(new Comment(1, "Max B", "Hello y'all!", new Date(1595320478000L)));
```

Comment:
I would recommend using the builder throughout and. make the Comment constructor private as per the design pattern

Reply:
https://www.geeksforgeeks.org/builder-design-pattern/#:~:text=PRG)%20Design%20Pattern-,Builder%20Design%20Pattern,step%20will%20return%20the%20object.